connectors/src/connectors/google_drive/index.ts

```diff
@@ -777,7 +787,8 @@ export class GoogleDriveConnectorManager extends BaseConnectorManager<null> {
         });
         const workflowRes = await launchGoogleDriveFullSyncWorkflow(
           this.connectorId,
-          null
+          null,
+          []
```

Comment:
I followed the existing pattern of passing null explicitly and not set default values.